### PR TITLE
Fix Bluetooth programing order

### PR DESCRIPTION
**Pull request by jcor25**
moved restarting on different baud rate after baud rate change command is sent, also commented set default out, if reset to default wanted rescanning for baud rate needed after reset to default.

### CODE REVIEW

**Comment by jcor25** [comment]
Verified working on JDY-31 and JDY-33.

File: firmware/console/binary/bluetooth.cpp [hunk 1]
```diff
@@ -108,11 +108,11 @@ static int btVerOk(SerialTsChannelBase* tsChannel)
 {
 	int len;
 	int ret = -1;
-	char tmp[38];
+	char tmp[46];
 
 	/* wait for resposne  */
 	len = btReadLine(tsChannel, tmp, sizeof(tmp));
-	if ((len >= 36) && (len <= 40)) {
+	if ((len >= 36) && (len <= 46)) {
```
Comment (scfrontier) [inline firmware/console/binary/bluetooth.cpp:115]: Hi Jcor25, what did you discover here?

Comment (jcor25) [reply firmware/console/binary/bluetooth.cpp:115]: this is for JDY-31 only. jdy31 do not respond to AT or AT+DISCOVERY, only AT+VERSION, update is for the newer batches that have longer response. now that I'm thinking about it you can also check for a known response such as AT+BAUD, response wound be AT+BAUD(?) so same amount of characters.

**Comment by scfrontier** [comment]
I think jcor23's solution is simpler than mine and solves the bug. If removing AT+DEFAULT creates problems for other users we can re-evaluate finding the "working" baud.

**Pull request closed without merging**